CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * The database schema version is now checked at boot, as well as in the http health checks. [#893](https://github.com/remind101/empire/pull/893)
 * The log level within empire can now be configured when starting the service. [#929](https://github.com/remind101/empire/issues/929)
 * The CloudFormation backend now has experimental support for a `Custom::ECSTaskDefinition` resource that greatly reduces the size of generated templates. [#935](https://github.com/remind101/empire/pull/935)
+* `emp run` now publishes an event when it is ran
```

Comment:
By convention, we also include a link to the PR that introduced the change (this one).